src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -32,29 +32,48 @@ public class StaticAnalyzer extends Scanner {
         
         private boolean m_openSourceOnly;
         private boolean m_sourceCodeOnly;
+        private String m_scanSpeed;
         
         @Deprecated
         public StaticAnalyzer(String target){
             this(target,false);
         }
         
-        public StaticAnalyzer(String target, boolean hasOptions, boolean openSourceOnly, boolean sourceCodeOnly){
+        public StaticAnalyzer(String target, boolean hasOptions, boolean openSourceOnly, boolean sourceCodeOnly, String scanSpeed){
             super(target, hasOptions);
             m_openSourceOnly=openSourceOnly;
             m_sourceCodeOnly=sourceCodeOnly;
+            m_scanSpeed=scanSpeed;
         }
         
 	@DataBoundConstructor
 	public StaticAnalyzer(String target,boolean hasOptions) {
 		super(target, hasOptions);
                 m_openSourceOnly=false;
                 m_sourceCodeOnly=false;
+                m_scanSpeed=DEEP;
```

Comment:
We shouldn't need to set the scan to deep, since that's the default.

Reply:
Okay, Matt. Made the required changes.